src/applications/toe/components/IntroductionLogin.jsx

```diff
@@ -18,15 +18,12 @@ function IntroductionLogin({
   isLOA3,
   isPersonalInfoFetchComplete,
   isPersonalInfoFetchFailed,
-  isSponsorsFetchComplete,
```

Comment:
Are these changes related to the PR? I'm having trouble tracking down where we use this in Pensions

Reply:
No, that is an unrelated change based on a previous base. That change should no longer be included in the file changes.